semcore/notice/src/style/notice.shadow.css

```diff
@@ -126,3 +126,16 @@ SNotice[theme='custom'] {
   background-color: var(--backgroundColor);
   border-color: var(--backgroundColor);
 }
+
+STitle {
+  font-size: var(--intergalactic-fs-300, 16px);
+  font-weight: var(--intergalactic-bold, 700);
+  line-height: var(--intergalactic-lh-300, 150%);
+  margin-top: 2px;
+  margin-bottom: 2px;
```

Comment:
As I see in figma - there is no margin-bottom.
for top - use please `var(--intergalactic-spacing-05x, 2px);`

Reply:
Updated. 

I took exact text styles from Figma, however my notice example still didn't match to what we have in examples. Title and icons in label and close button looked misaligned. So i have to also take styles from examples too.

Reply:
done